backend/src/docs/v1.api-spec.yaml

```diff
@@ -268,6 +270,12 @@ components:
             type: string
             example: CMSG
           description: An array of applicationAcronyms
+        passwordPublicKey:
+          type: string
+          example: MIGfMA0GCSqGS...
```

Comment:
It may be worth putting in a description for this - public keys can be represented in many ways (ie PEM, PKCS12, or whatever other formats exist). We want to make sure that whomever looks at the API definition can understand without a doubt what the data format should be without having to look at implementation details or the library we used to achieve it.